src/e2e-test/features/Pipeline.feature

```diff
@@ -0,0 +1,50 @@
+#
```

Comment:
I think it would be better to rename the file to SanityTests.feature to reflect the intent